src/webui/api/appcontroller.cpp

```diff
@@ -68,6 +69,18 @@ void AppController::versionAction()
     setResult(QBT_VERSION);
 }
 
+void AppController::buildInfoAction()
+{
+    const QJsonObject versions = {
+        {"qt", QT_VERSION_STR},
+        {"libtorrent", Utils::Misc::libtorrentVersionString()},
+        {"boost", Utils::Misc::boostVersionString()},
+        {"openssl", Utils::Misc::opensslVersionString()},
```

Comment:
BTW, why don't we mention one more direct dependency library, ZLib?
(GUI too)

Reply:
Created issue #10240